### DISCUSSION TITLE
Invalid chars are inserted in code with latest version 0.16.0

### DISCUSSION THREAD

**Discussion by alokc83** [intitni/CopilotForXcode#224] [General]
Not sure if anyone else is seeing this. 
Only reject button dismiss the suggestion. Accept button does not add suggestions to the code but add invalid hidden char in the code. 
Downgraded to 0.15.2 
<img width="255" alt="Screenshot 2023-05-22 at 8 44 06 PM" src="https://github.com/intitni/CopilotForXcode/assets/59144/95dd0015-2809-473d-98f0-bba5b47ec003">

**Comment by intitni** [comment]
Please open an issue and describe how you triggered the accept button. Did you use the UI, keybindings, or menu? It would be great if you could let me know whether these three methods result in different outcomes.

**Comment by intitni** [comment]
Maybe related #137
